samples/dfu: moved smp_svr_mini_boot sample here (#27936)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

DFU samples
-----------

* Updated:

>   * ref:`_mcuboot_minimal_configuration` moved under ``samples/dfu`` directory.

Reviewer: ```suggestion
  * The :ref:`_mcuboot_minimal_configuration` has been moved to the :file:`samples/dfu` directory.
```

>   * ref:`_mcuboot_minimal_configuration` moved under ``samples/dfu`` directory.

Reviewer 2: ```suggestion
  * The :ref:`mcuboot_minimal_configuration` has been moved to the :file:`samples/dfu` directory.
```
